plugins/@grouparoo/redshift/__tests__/integration/redshift-query-import.ts

```diff
@@ -0,0 +1,152 @@
+import path from "path";
```

Comment:
These integration tests are modified copies of the versions in the Postgres plugin